compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/ConversionUtils.kt

```diff
@@ -406,6 +406,9 @@ fun isOverriding(
             // Not checking the field type (they should match each other if everything other match, otherwise it's a compilation error)
             target.name == superCandidate.name
         }
+        target is IrProperty && superCandidate is IrProperty -> {
```

Comment:
Perhaps redundant, but just to make sure:
```
// Not checking the property return type (they should match each other if everything other match, otherwise it's a compilation error)
```
Is that right? Then, it'd be good to explicitly mention that as a comment. :)

Reply:
done